README.md

```diff
@@ -28,3 +28,40 @@ The build is minified and the filenames include the hashes.<br />
 Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+## Custom Components/Services:
```

Comment:
please move this to Documentation/Alerts directory and add here a link to the file.